backend/APIFiles/quickBuy.py

```diff
@@ -0,0 +1,68 @@
+import os
+from fastapi import FastAPI
+from supabase import create_client, Client
+from dotenv import load_dotenv
+from datetime import datetime, timedelta
+from APIFiles import testData
+from APIFiles import supabaseMiddleman
+
+
+app=FastAPI()
+load_dotenv()
+
+url = os.getenv('PUBLIC_SUPABASE_URL')
+key = os.getenv('SUPABASE_KEY')
+supabase: Client = create_client(url, key)
+
+
+async def buyMethod():
+    # Return state by looking for the one with the biggest ID
+    isOpen = supabaseMiddleman.isMarketOpen()
+    if isOpen:
```

Comment:
```suggestion
    if not isOpen:
        return "The market is closed"
```

and deindent the rest of the function. Same with the other check. Otherwise the link between if/else gets lost and the code gets unreadable fast